client/src/scene/SceneGameArena.ts

```diff
@@ -177,13 +198,23 @@ export class SceneGameArena extends Phaser.Scene {
     }
 
     private updateDrawPlayers() {
-        this.gameState.currentTetromino.draw(this);
+        if (
+            this.gameState.playerId !== null &&
+            this.gameState.playerId !== undefined
```

Comment:
Just as a note, the single `this.gameState.playerId != null` would account for both situations. I think you've used this in multiple places but its up to you if you want to swap them over.